horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Sorry.
> Is there a way to exclude blocks of code from certain checks?

I see only to exclude the complete file in the docs.
--exclude=patterns   exclude files or directories
https://pycodestyle.readthedocs.io/en/latest/intro.html#configuration

> I prefer the original layout, especially when lots of lines are aligned in a tabular-like style.

For my opinion is the tabular-like style good for to look fast what is happen a row under, but in this it is not important, what values in the other row. On the contrary, it brings the reader confused because he believes in the first moment that would means something. But each line stands for itself.